build-runtime.sh.in

```diff
@@ -1,19 +1,31 @@
 #! /bin/bash
 
+########################
+# build runtime script #
+########################
+#
+# template to be configured by the CMake build system
+
 if which clang >/dev/null; then
     CC=clang
 else
     CC=gcc
 fi
 
-set -x
+STRIP=strip
+
+set -xe
 
 build_dir=$(mktemp -d)
 orig_cwd=$(pwd)
-repo_root=$(dirname $(readlink -f $0))
+repo_root="@PROJECT_SOURCE_DIR@"
```

Comment:
What is the advantage of doing it this way? I like the original more because everyone can understand what it means whereas it is unclear what `PROJECT_SOURCE_DIR` happens to be.